tests/member_chain/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -49,12 +49,35 @@ $weNeedToReachTheEightyCharacterLimitXXXXXXXXXXXXXXXXX->someNode
 ->childrenInAnArray[0];
 $superSupersuperSupersuperSupersuperSupersuperSuperLong->exampleOfOrderOfGetterAndSetterReordered;
 $superSupersuperSupersuperSupersuperSupersuperSuperLong->exampleOfOrderOfGetterAndSetterReordered[0];
+$superSupersuperSupersuperSupersuperSupersuperSuperLong->{$exampleOfOrderOfGetterAndSetterReordered};
+$superSupersuperSupersuperSupersuperSupersuperSuperLong::$exampleOfOrderOfGetterAndSetterReordered;
+$superSupersuperSupersuperSupersuperSupersuperSuperLong::$exampleOfOrderOfGetterAndSetterReordered[0];
+$superSupersuperSupersuperSupersuperSupersuperSuperLong = (new SuperSuperSuperSuperSuperSuperLongClassName())->some_very_long_member_expression;
+new $superSupersuperSupersuperSupersuperSupersuperSuperLong->superSupersuperSupersuperSupersuperSupersuperSuperLong->superSupersuperSupersuperSupersuperSupersuperSuperLong();
+(new $superSupersuperSupersuperSupersuperSupersuperSuperLong->superSupersuperSupersuperSupersuperSupersuperSuperLong->superSupersuperSupersuperSupersuperSupersuperSuperLong)->superSupersuperSupersuperSupersuperSupersuperSuperLong;
+$test = $superSupersuperSupersuperSupersuperSupersuperSuperLong->exampleOfOrderOfGetterAndSetterReordered;
+$test = $test = $superSupersuperSupersuperSupersuperSupersuperSuperLong->exampleOfOrderOfGetterAndSetterReordered;
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 <?php
-$SomeVeryLongUpperCaseConstant->someVeryLongCallExpression()->some_very_long_member_expression;
-$weNeedToReachTheEightyCharacterLimitXXXXXXXXXXXXXXXXX->someNode->childrenInAnArray[0];
+$SomeVeryLongUpperCaseConstant->someVeryLongCallExpression()
+    ->some_very_long_member_expression;
+$weNeedToReachTheEightyCharacterLimitXXXXXXXXXXXXXXXXX->someNode
+    ->childrenInAnArray[0];
 $superSupersuperSupersuperSupersuperSupersuperSuperLong->exampleOfOrderOfGetterAndSetterReordered;
-$superSupersuperSupersuperSupersuperSupersuperSuperLong->exampleOfOrderOfGetterAndSetterReordered[0];
+$superSupersuperSupersuperSupersuperSupersuperSuperLong
+    ->exampleOfOrderOfGetterAndSetterReordered[0];
+$superSupersuperSupersuperSupersuperSupersuperSuperLong->{$exampleOfOrderOfGetterAndSetterReordered};
+$superSupersuperSupersuperSupersuperSupersuperSuperLong::$exampleOfOrderOfGetterAndSetterReordered;
+$superSupersuperSupersuperSupersuperSupersuperSuperLong::$exampleOfOrderOfGetterAndSetterReordered[0];
+$superSupersuperSupersuperSupersuperSupersuperSuperLong = (new SuperSuperSuperSuperSuperSuperLongClassName())
```

Comment:
shouldn't this break at the `=` first?

Reply:
@mgrip js prettier don't break, i do same logic, we can change this if a lot of people vote for break